ingestion_server/test/unit_tests/test_distributed_reindex_scheduler.py

```diff
@@ -54,19 +61,8 @@ def test_assign_work(estimated_records, record_limit, workers, expected_ranges):
             "sample_index",
         )
 
-        for worker, mock_post, (start_id, end_id) in zip(
-            workers, post_mocks, expected_ranges
-        ):
-            assert mock_post.calls > 0
-            data = json.loads(mock_post.matches[0].body)
-            print(f"{data=}")
-            assert data == {
-                "model_name": "sample_model",
-                "table_name": "sample_table",
-                "target_index": "sample_index",
-                "start_id": start_id,
-                "end_id": end_id,
-            }
+        # Pook will raise an exception here if any requests don't match the above
```

Comment:
Does pook raise the exception? I think it just returns the boolean :thinking:

Reply:
Sorry, you're right - I'll reword this!